epoptes/ui/gui.py

```diff
@@ -963,7 +891,7 @@ def iconsSizeScaleChanged(self, widget=None, width=None):
         self.scrHeight = int(3 * self.scrWidth / 4) # Κeep the 4:3 aspect ratio
 
         # Fast scale all the thumbnails to make the change quickly visible
-        old_pixbufs = self.imagetypes.values()
+        old_pixbufs = list(self.imagetypes.values())
```

Comment:
Why list()?

Reply:
Thanks, they were [automatically added by 2to3](https://stackoverflow.com/questions/27476079/why-does-2to3-change-mydict-keys-to-listmydict-keys), I removed them in https://github.com/eellak/gsoc2018-epoptes/pull/8/commits/ddbbd59aa4bb9c16e22fa326508c99506a0013b5.